tools/wptrunner/wptrunner/executors/executorcontentshell.py

```diff
@@ -276,6 +287,13 @@ def do_test(self, test):
                 self.logger.warning('URL mismatch may be a false positive '
                                     'if the test navigates')
                 result_url = test.url
+
+            if self.protocol.content_shell_test.leaked:
+                test.restart_test_after = True
+                if status == 0:
```

Comment:
Use constants for 0/1 instead of hardcoded magic numbers? Then we don't need the comment to explain what's going on.

Reply:
moved the status logic to appropriate place and added some comments. Should be more clear though since the status_code map is in same place.
The test.restart_test_after should probably stay here because we need to revise the "test" object in order to force a restart.